modules/income_and_assets/spec/lib/income_and_assets/pdf_fill/va21p0969_spec.rb

```diff
@@ -15,13 +15,15 @@ def basic_class
     VetsJsonSchema::EXAMPLES.fetch('21P-0969-KITCHEN_SINK')
   end
 
-  it_behaves_like 'a form filler', {
-    form_id: described_class::FORM_ID,
-    factory: :income_and_assets_claim,
-    use_vets_json_schema: true,
-    input_data_fixture_dir: 'modules/income_and_assets/spec/fixtures/pdf_fill/21P-0969',
-    output_pdf_fixture_dir: 'modules/income_and_assets/spec/fixtures/pdf_fill/21P-0969'
-  }
+  # TODO: Once all of the sections have been merged, regenerate the PDF fixtures and
+  # reenable this test (captured in ticket #106962)
```

Comment:
Will re-enable as part of the work for this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/106962